src/scrape_up/github/users.py

```diff
@@ -409,6 +514,13 @@ def get_repository_details(self):
                         }
                     )
 
-            return repositories
+            return {
+                "data": repositories,
+                "message":f"Repositories found for user {self.username}"
+            }
         except:
-            return "No repositories found"
+            message = f"{self.username} not found !"
+            return {
+                "data":None,
+                "message":message
```

Comment:
Change the message according to the function.

Also, make the changes for all the classes, not only `User`